easybuild/easyconfigs/f/flex/flex-2.5.39-iomkl-2016.07.eb

```diff
@@ -0,0 +1,14 @@
+name = 'flex'
+version = '2.5.39'
+
+homepage = 'http://flex.sourceforge.net/'
+description = """Flex (Fast Lexical Analyzer) is a tool for generating scanners. A scanner,
+ sometimes called a tokenizer, is a program which recognizes lexical patterns in text."""
+
+toolchain = {'name': 'iomkl', 'version': '2016.07'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://prdownloads.sourceforge.net/%(namelower)s']
+
```

Comment:
M4 is a dep for flex 2.5.39 (cfr. #3678), see #3656 for up-to-date deps for flex 2.6.0